lib/action_controller/serialization/context.rb

```diff
@@ -0,0 +1,18 @@
+module ActionController
+  module Serialization
+    class Context
+      include Rails.application.routes.url_helpers if Rails.application.present?
+
+      def self.default_url_options
+        ActionMailer::Base.default_url_options
+      end
```

Comment:
I'd rather not include the url helpers and instead stick 'em in a url_helpers method

We could probably set it and default_url_iptions in the railtie.

Also, i don't believe we include action mailer nor do we need it. There should be a similar methid on the controller